src/persistence/db.js

```diff
@@ -3,6 +3,7 @@ import fs from 'fs'
 import path from 'path'
 import { getDbPath } from './db_util'
 
+const MixinDatabaseVersion = 3
```

Comment:
3 ？ it's from 0 default.

Reply:
I saw before MixinDatabaseVersion  is 2, can i use 0.1 now?

Reply:
set to 1